.github/move.yml

```diff
@@ -16,6 +16,6 @@ mentionAuthors: true
 keepContentMentions: true
 
 # Set custom aliases for targets
-# aliases:
-#   r: repo
-#   or: owner/repo
+aliases:
+  flutter: flutter/flutter
+  flutter: flutter/flutter-intellij
```

Comment:
I'm new to the syntax but it seems a little odd to have one alias for two targets.  Is that right?

Reply:
No, good catch - the 2nd one should be `flutter-intellij`.